### PR TITLE
Make thanepark use Rides instead of persons

### PR DESCRIPTION
**Pull request by keenlok**
#25 #26 #28

### CODE REVIEW

File: src/test/java/systemtests/HelpCommandSystemTest.java [hunk 1]
```diff
@@ -65,9 +65,9 @@ public void openHelpWindow() {
         assertEquals("", getCommandBox().getInput());
         assertCommandBoxShowsDefaultStyle();
         assertNotEquals(HelpCommand.SHOWING_HELP_MESSAGE, getResultDisplay().getText());
-        assertNotEquals(BrowserPanel.DEFAULT_PAGE, getBrowserPanel().getLoadedUrl());
-        assertListMatching(getPersonListPanel(), getModel().getFilteredRideList());
-
+        assertNotEquals(getClass().getResource(HelpWindow.SHORT_HELP_FILE_PATH), getBrowserPanel().getLoadedUrl());
+        assertListMatching(getPersonListPanel(), getModel().getFilteredPersonList());
+
```
Comment (YuPeiHenry) [inline src/test/java/systemtests/HelpCommandSystemTest.java]: Trailing whitespace

Comment (keenlok) [reply src/test/java/systemtests/HelpCommandSystemTest.java]: thanks

**Comment by keenlok** [comment]
not sure if there is still any remaining persons in the file

File: src/test/java/seedu/address/model/ride/RideContainsKeywordsPredicateTest.java [hunk 2]
```diff
@@ -42,19 +42,25 @@ public void equals() {
...
         // Only one matching keyword
-        predicate = new RideContainsKeywordsPredicate(Arrays.asList("Bob", "Carol"));
-        assertTrue(predicate.test(new RideBuilder().withName("Alice Carol").build()));
+        predicate = new RideContainsKeywordsPredicate(Arrays.asList("Big", "Castle"));
+        assertTrue(predicate.test(new RideBuilder().withName("Alice Castle").build()));
 
         // Mixed-case keywords
         predicate = new RideContainsKeywordsPredicate(Arrays.asList("aLIce", "bOB"));
         assertTrue(predicate.test(new RideBuilder().withName("Alice Bob").build()));
+
```
Comment (YuPeiHenry) [inline src/test/java/seedu/address/model/ride/RideContainsKeywordsPredicateTest.java]: Trailing whitespace

File: src/test/java/seedu/address/model/ride/RideContainsKeywordsPredicateTest.java [hunk 3]
```diff
@@ -67,9 +73,6 @@ public void test_nameDoesNotContainKeywords_returnsFalse() {
         predicate = new RideContainsKeywordsPredicate(Arrays.asList("Carol"));
         assertFalse(predicate.test(new RideBuilder().withName("Alice Bob").build()));
 
-        // Keywords match days since last maintenance, current waiting time and address, but does not match name
-        predicate = new RideContainsKeywordsPredicate(Arrays.asList("12345", "alice@email.com", "Main", "Street"));
-        assertFalse(predicate.test(new RideBuilder().withName("Alice").withMaintenance("12345")
-                .withWaitTime("1").withAddress("Main Street").build()));
+
```
Comment (YuPeiHenry) [inline src/test/java/seedu/address/model/ride/RideContainsKeywordsPredicateTest.java]: Trailing whitespace

**Pull request merged**